### PR TITLE
Fix error on button size example

### PR DESCRIPTION
**Pull request by donnisnoni**
https://user-images.githubusercontent.com/16399020/105905321-432a4200-605d-11eb-9912-560dca9afbaf.mp4

### CODE REVIEW

File: packages/docs/src/playground/Button.svelte [hunk 1]
```diff
@@ -30,7 +30,7 @@
     block={values.block}
     disabled={values.disabled}
     active={values.active}
-    size={values.size}
+    size={(values.size = values.size && values.size.length ? values.size : 'default')}
```
[the remark this answers is not in the document]
Comment (TheComputerM) [reply packages/docs/src/playground/Button.svelte:33]: Can it be replaced by:
```js
values.size || 'default'
```

[the remark this answers is not in the document]
Comment (donnisnoni) [reply packages/docs/src/playground/Button.svelte:33]: Wait... I think I use that solution earlier. The problem is the text field will be also empty.
That's why I'm using 

```js
(values.size = values.size && values.size.length ? values.size : 'default')
```

**Comment by TheComputerM** [comment]
Thanks :+1:

File: packages/docs/src/playground/Button.svelte [hunk 2]
```diff
@@ -30,7 +31,7 @@
     block={values.block}
     disabled={values.disabled}
     active={values.active}
-    size={(values.size = values.size || 'default')}
+    size={values.size}
```
Comment (donnisnoni) [inline packages/docs/src/playground/Button.svelte:34]: Ahhh.... got it 😁

Comment (TheComputerM) [reply packages/docs/src/playground/Button.svelte:34]: I passed in the mandatory option to the select so that values.size cannot be falsy

**Pull request merged**